tests/test_digest_handler.py

```
@@ -27,8 +49,9 @@ def __init__(self, response: str = "stub digest body") -> None:
         self.response = response
         self.calls: list[tuple[str, str]] = []
 
-    def call(self, system_prompt: str, user_prompt: str,
-             max_tokens: int = 0) -> str:
+    def call(
+        self, system_prompt: str, user_prompt: str, max_tokens: int = 0
```

Ruff (0.15.12)

[warning] Unused method argument: `max_tokens`

(ARG002)